Projects/Dotmim.Sync.Core/Dm/DmTable.cs

```diff
@@ -807,13 +807,15 @@ public void Fill(IDataReader reader)
             if (readerFieldCount < this.columns.Count - 2)
                 return;
 
-            while (reader.Read())
+            if (reader.NextResult())
```

Comment:
ping @VladislavAntonyuk

Reply:
Agree. It can be used like
```
do
        {
            SomeAction(reader);
        } while (reader.NextResult());
```

I will replace IDataReader  with DbDataReader, so it will be used everywhere in project.

Reply:
I have a question about lines 806-819. Do we need them? What value do we get from that code?

Reply:
Yes, we need it, because the instruction `this.LoadDataRow(readerDataValues, true);` will add the values from the reader to the `Rows` property.
Eventually, something that could be fixed would be replacing `var dataRow = this.LoadDataRow(readerDataValues, true);` with `this.LoadDataRow(readerDataValues, true);`